src/views/UserInfoWrapper/index.tsx

```diff
@@ -34,7 +34,7 @@ const UserInfoWrapper = ({ children }: UserInfoWrapperProps) => {
       const user = {
         name: oktaUser.name,
         euaId: oktaUser.euaId || '',
-        groups: oktaUser.groups || [],
+        groups: oktaUser['mint-groups'] || [],
```

Comment:
The local `getUser` method on `oktaAuth` returns `groups`, the `authState` returns `mint-groups`.